seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/file/ClickhouseFileSinkWriter.java

```diff
@@ -270,8 +241,7 @@ private List<String> generateClickhouseLocalFiles(String clickhouseLocalFileTmpF
         }
         command.add("--file");
         command.add(clickhouseLocalFileTmpFile);
-        command.add("--format_csv_delimiter");
-        command.add("\"" + readerOption.getFileFieldsDelimiter() + "\"");
+        command.add(" --input-format Avro");
```

Comment:
@caicancai Can it be made optional here? For example, supporting CSV and AVRO, etc

Reply:
There are many bugs in csv format writing. From my experience, if you want to use csv, you must write a special parse for csv, such as rowconvertTocsv. I am not sure whether you still need to keep csv.

cc @Hisoka-X